sky/engine/core/dart/window.dart

```diff
@@ -18,41 +24,102 @@ class WindowPadding {
   final double left;
 }
 
+/// An identifier used to select a user's language and formatting preferences,
+/// consisting of a language and a country. This is a subset of locale
+/// identifiers as defined by BCP 47.
 class Locale {
-  Locale(this.languageCode, this.countryCode);
+  const Locale(this.languageCode, this.countryCode);
 
+  /// The primary language subtag for the locale.
   final String languageCode;
+
+  /// The region subtag for the locale.
   final String countryCode;
 
+  bool operator ==(dynamic other) {
```

Comment:
Quiver was never going to help with the `operator ==`s. I don't think there's anything we can really do about those. At least we've established a pattern, though.

For the hashCodes, I am working on a better solution, see #88.